routemaster/lua/nack.lua

```diff
@@ -0,0 +1,12 @@
+--
+-- Negative-acknowledges a message.
+--
+-- KEYS[1]: list of new message UUIDs
+-- KEYS[2]: zset of pending (nacked) message UUIDs
```

Comment:
Would it be worth putting the zset argument first for parity with the order of the arguments in the `ack` script? It's not really a big deal though.